package.json

```diff
@@ -53,7 +53,7 @@
     "build:src": "npm run lint && npm run build:fast && npm run build:test && npm run build:doc && npm run build:completion",
     "build:fast": "gulp pages && tsc && npx oclif-dev manifest",
     "build:test": "tsc -P ./tsconfig.dev.json --noEmit && tsc -P ./tsconfig.js.json --noEmit",
-    "build:doc": "ts-node --transpile-only automation/capitanodoc/index.ts > doc/cli.markdown",
+    "build:doc": "ts-node --transpile-only automation/capitanodoc/index.ts | tee doc/cli.markdown docs/balena-cli.md",
```

Comment:
`tee` may be available on MSYS (the build tests passed!) but would not be available on PowerShell and `cmd.exe` (I assume). Reason why we use `rimraf` instead of `rm -rf`. Not sure how much it matters. However if you adopt my suggestion in another comment to avoid markdown duplication/copy, we avoid the need for `tee` here as well.